docs/auth/identity-resolver.md

```diff
@@ -12,6 +12,32 @@ If you want to use an auth provider to sign in users, you need to explicitly con
 it have sign-in enabled and also tell it how the external identities should
 be mapped to user identities within Backstage.
 
+## Guest Sign-In Resolver
+
+Backstage projects created with `npx @backstage/create-app` come configured with a
+sign-in resolver for GitHub guest access. This resolver makes all users share
+a single "guest" identity and is only intended as a minimum requirement to quickly
+get up and running. You can replace `github` for any of the other providers if you need.
+
+This resolver should not be used in production, as it uses a single shared identity,
+and has no restrictions on who is able to sign-in. Be sure to read through the rest
+of this page to understand the Backstage identity system once you need to install
+a resolver for your production environment.
+
+The guest resolver can be useful for testing purposes too, and it looks like this:
+
+```ts
+signIn: {
+  resolver: (_, ctx) =>
+    ctx.issueToken({
```

Comment:
Even though the curly braces around this one might be unnecessary, it may be useful to have them in there anyway since the user will probably want to expand on it eventually and this helps them get going (same below). Maximizing the "just start typing" factor of it :)